crates/bevy_render/src/camera/camera.rs

```diff
@@ -96,6 +96,16 @@ pub struct ExposureSettings {
 }
 
 impl ExposureSettings {
+    pub const SUNLIGHT: Self = Self {
+        ev100: Self::EV100_SUNLIGHT,
+    };
+    pub const OVERCAST: Self = Self {
+        ev100: Self::EV100_OVERCAST,
+    };
+    pub const INDOOR: Self = Self {
+        ev100: Self::EV100_INDOOR,
+    };
+
     pub const EV100_SUNLIGHT: f32 = 15.0;
     pub const EV100_OVERCAST: f32 = 12.0;
     pub const EV100_INDOOR: f32 = 7.0;
```

Comment:
Might be worth creating several constants for indoor lighting. The linked wikipedia article says:

Sports events, stage shows, and the like = 8–9
Offices and work areas = 7–8
Home interiors = 5–7

Reply:
I kind of like the current granularity. Something like "office" feels too specific to be correct. All of these will be +- 2 or 3. Anyone manually setting exposure will likely be doing it artistically / without exact constants anyway.